amd_openvx/openvx/CMakeLists.txt

```diff
@@ -104,18 +94,12 @@ elseif (GPU_SUPPORT AND "${BACKEND}" STREQUAL "HIP")
         message(STATUS "Found HIP: " ${HIP_PATH} " version: " ${HIP_VERSION})
         if(HIP_COMPILER STREQUAL clang)
             add_subdirectory(hipvx)
-            add_library(vxu SHARED api/vxu.cpp)
-            set(ENABLE_OPENCL 0)
             set(ENABLE_HIP 1)
-            target_compile_definitions(vxu PUBLIC ENABLE_OPENCL=${ENABLE_OPENCL})
-            target_compile_definitions(vxu PUBLIC ENABLE_HIP=${ENABLE_HIP})
-            add_library(openvx SHARED ${SOURCES} $<TARGET_OBJECTS:openvx_hip>)
+            target_sources(openvx $<TARGET_OBJECTS:openvx_hip>)
```

Comment:
@kiritigowda this line causes a CMake error (see [here](http://math-ci.rocm.amd.com/blue/organizations/jenkins/compute-rocm-dkms-no-npi-hipclang%2FShort-GPUOpen%2FMIVisionX/detail/PR-844/2/pipeline/#step-255-log-1832)), please double-check.

CMake Error at amd_openvx/openvx/CMakeLists.txt:98 (target_sources):
target_sources called with invalid arguments